lib/spack/spack/test/conftest.py

```diff
@@ -1214,6 +1214,16 @@ def mock_fetch(mock_archive, monkeypatch):
     )
 
 
+@pytest.fixture()
```

Comment:
windows tests are barfing on this one. @johnwparent can you advise?
@matthewcurry perhaps move this fixture local to the file it is used for now and apply `@pytest.mark.not_on_windows("can't package fixture on windows")`?

Reply:
Oh ok, this is pretty straightforward. Windows uses `spawn` to launch `multiprocessing` subprocesses, which utilizes a fresh interp, and thus requires pickling to transfer state. Local objects cannot be pickled.